ModLoader/ModLoader.cpp

```diff
@@ -164,8 +164,8 @@ void commonItems::ModLoader::loadModDirectories(const std::vector<std::string>&
 					Log(LogLevel::Warning) << "\t\tError while reading " << metadata_location << "! Mod will not be useable for conversions.";
 					continue;
 				}
-				possibleUncompressedMods.emplace_back(Mod(theMod.getName(), theMod.getPath(), theMod.getDependencies(), theMod.getReplacedPaths()));
-				Log(LogLevel::Info) << "\t\tFound a potential mod [" << theMod.getName() << "] at " << theMod.getPath();
+				processLoadedMod(theMod, mod.name, mod_folder, mod.path, gameModPath);
```

Comment:
This will bring back the 
```Log(LogLevel::Info) << "\t\tFound a potential mod [" << theMod.getName() << "] with a mod file at " << path << " and itself at " << theMod.getPath();```
message in fileUnderCategory(), which means meta-mods get logged twice. And with the mod file listed at a ver nonsensical location. That's why I bypassed the function in the first place. The checks weren't relevant either.